backend/src/services/notification/notification.service.ts

```diff
@@ -1,7 +1,45 @@
-import { logger } from '~/services/services';
+import { NotificationKey } from '~/common/enums';
+import { GeolocationKey, MapKey } from '~/common/enums';
+import { geolocation as geolocationRepository, notification as notificationRepository } from '~/data/repositories';
+import { IGeolocation, ICoordsSet, INewNotification } from '~/common/interfaces';
+
 class Notification {
-  public sendNotifications(): void {
-    logger.log('Coords received');
+  public async sendNotifications(selectedArea: ICoordsSet): Promise<void> {
+    const relativeGeolocations = await this.getRelativeGeolocations(selectedArea);
+    const relativeUserIds = this.getRelativeUserIds(relativeGeolocations);
+    const fromUserId = selectedArea.userId;
+
+    relativeUserIds.forEach(userId => {
+      const newNotification: INewNotification = {
+        [NotificationKey.TEXT]: 'Please, stay home if possible.',
+        [NotificationKey.TOPIC]: 'COVID-19!',
+        [NotificationKey.TO]: userId as string,
+        [NotificationKey.USER_ID]: fromUserId as string,
+      };
+
+      notificationRepository.createNotification(newNotification);
+    });
+  }
+
+  private async getRelativeGeolocations(selectedArea: ICoordsSet): Promise<IGeolocation[]> {
```

Comment:
```suggestion
  private async checkIsRelativeGeolocations(selectedArea: ICoordsSet): Promise<IGeolocation[]> {
```
Move it to `backend/src/services/notification/helpers`